homeworks/08-frp/webpack.config.js

```diff
@@ -0,0 +1,54 @@
+const webpack = require('webpack')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const CopyPlugin = require("copy-webpack-plugin");
+const path = require('path')
+
+module.exports = (env, argv) => {
+  return {
+    mode: argv.mode || 'none',
+    entry: './src/index.ts',
+    output: {
+      path: path.resolve(__dirname, 'dist'),
+    },
+    resolve: {
+      extensions: ['.js', '.ts'],
+    },
+    module: {
+      rules: [
+        {
+          test: /\.ts$/,
+          use: {
+            loader: 'babel-loader',
+            options: {
+              presets: [
+                '@babel/preset-typescript',
+              ],
+              plugins: [
+                ['@babel/plugin-transform-runtime'],
+              ],
+            },
+          },
+        },
+        {
+          test: /\.css$/i,
+          use: ['style-loader', 'css-loader'],
+        },
+        {
+          test: /\.svg$/,
+          loader: 'svg-inline-loader',
+        },
+      ],
+    },
+    plugins: [
+      new webpack.ProgressPlugin(),
+      new HtmlWebpackPlugin({
+        template: './src/index.html',
+      }),
+      new CopyPlugin({
+          patterns: [
+              { from: "images/*.svg", to(_) { return "./images/[name][ext]" } }
```

Comment:
We haven't learned wepback yet, so it's not critical at all but in this case it's better to use native webpack loader as:
```javascript
{
  ...,
  module: {
    rules: [
      ...,
     {
        test: /\.svg$/,
        type: 'asset/inline',
     },
  },
}
```

And then you can import svg directly in your code:

```javascript
import cat from './cat.svg';

const cat$ = from(loadImage(cat));
```